resources/LoggingBar.qml

```diff
@@ -287,11 +286,14 @@ Rectangle {
             repeat: true
             onTriggered: {
                 logging_bar_model.fill_data(loggingBarData);
-                sbpLoggingButton.checked = loggingBarData.sbp_logging;
-                csvLoggingButton.checked = loggingBarData.csv_logging;
+                if (sbpLoggingFormat.currentIndex == -1) {
+                    sbpLoggingFormat.currentIndex = loggingBarData.sbp_logging_format_index;
+                    sbpLoggingButton.checked = loggingBarData.sbp_logging;
+                    csvLoggingButton.checked = loggingBarData.csv_logging;
+                }
                 if (loggingBarData.recording_filename)
                     recordingFilenameText.editText = loggingBarData.recording_filename;
-                let recording = sbpLoggingButton.checked || csvLoggingButton.checked;
+                let recording = loggingBarData.sbp_logging || loggingBarData.csv_logging;
```

Comment:
I might be out of the loop on this but I'm not sure we want to mix these up. What happens if you record sbp and csv at the same time?